readme.md

```diff
@@ -113,19 +121,23 @@ $ npm -v
 
 * **Your Answer:**
 
+My Name
```

Comment:
But why? In this case, it's because we exported that value from our `./profile.js` file.